apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.CommonTesting/SpannerTestDatabasePostgreSql.cs

```diff
@@ -90,10 +90,11 @@ protected override void CreateDatabase()
             }
 
             Logger.DefaultLogger.Debug($"The database {SpannerDatabase} with dialect {dialect} created successfully.");
```

Comment:
We have the logger message here and we have removed it from the other class. Possibly keep the create database message in the derived classes as they are different and in case of reusing the database (i.e. when false is returned), we can log the message in the base class?